README.md

```diff
@@ -80,57 +84,57 @@ Checkout [tests](test) for more examples.
 
 #### `root`
 
-Type: `String`  
```

Comment:
The 2 spaces are intentional (to create new lines). Can you put them back ? Using editorconfig might helps to respect that (see https://github.com/postcss/postcss-import/blob/master/.editorconfig#L12-L13 )